example/server/integrations.py

```diff
@@ -586,7 +545,7 @@ def info(self, asset: Asset, lang: Optional[str] = None):
         }
 
     def process_send_request(self, params: Dict, transaction_id: str) -> Optional[Dict]:
-        sender_id = params.get("sender_id")  # not actually used
+        _ = params.get("sender_id")  # not actually used
```

Comment:
whats the purpose of this line?

Reply:
`_` is a way of not assigning a variable. I just have the line to show that we collect `sender_id`. We don't use the sender info when processing both those looking at the code for reference might.